### ISSUE TITLE
How to install a CI_package_check for ARM

### ISSUE THREAD

**Issue by yalh76** [YunoHost/CI_package_check#14]
I've looked and try different scripts but didn't find the way to install an ARM CI to be used using SSH from another CI_package_check...

**Comment by maniackcrudelis** [comment]
If you're looking for an ARM CI, used by a master CI by ssh, then you're looking for this script https://github.com/YunoHost/CI_package_check/blob/master/auto_build/chroot_ssh.sh

Then you have to configure your master CI in https://github.com/YunoHost/CI_package_check/blob/master/config.modele to add this slave.

**Comment by yalh76** [comment]
Well, My main problem was that package_check didn't built the lxc container on aarch64.
After solving that, I was able to install a CI using https://github.com/YunoHost/CI_package_check/blob/master/build_CI.sh

I just had another issue for ssh connection from a x86-64b CI to the ARM64/aarch64 CI: I had the error: `Warning: Identity file "/home/CI_package_check/pcheckCI_key" not accessible: No such file or directory`.
To solve that I removed the " in config file: `> 0.arm.ssh_key="/home/CI_package_check/pcheckCI_key"`
So now I'm able to launch package_checker for x86-64b and ssh ARM64.

The only point left is: I have 3 VMs on ARM64 with CI_package_check installed and functional.
But when I add the three ssh ARM CI in the config file, a CI test for ARM goes to the x86-64b instead of one of the ssh ARM CI
When I have only one ssh ARM CI in the config file, a CI test for ARM goes to the ssh ARM CI
But I'm not sure that's important, as far as I've seen, I can't launch multiple test at the same time, for example one on x86-64b and one on ssh ARM64. The first test need to be finished to launch the next one even if it's on another architecture
